cgmanifest.json

```diff
@@ -4930,8 +4930,8 @@
         "type": "other",
         "other": {
           "name": "gstreamer1",
-          "version": "1.20.0",
-          "downloadUrl": "http://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.20.0.tar.xz"
+          "version": "1.24.8",
+          "downloadUrl": "http://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.8.tar.xz"
```

Comment:
Please switch to `https`. Same for `Url:` & `Source0:` in spec file.

Reply:
Done